demos/system_reference-NXP.rst

```diff
@@ -8,6 +8,102 @@ NXP Reference Boards
 
 A number of the `OpenAMP project examples <https://github.com/OpenAMP/openamp-system-reference/tree/main/examples>`_ can be executed on NXP Reference boards.
 
-* NXP `8MPLUSLPD4-EVK <https://www.nxp.com/design/design-center/development-boards-and-designs/8MPLUSLPD4-EVK>`_.
+Board setup
+------------
 
-The `AN13970 Running Zephyr RTOS on Cadence Tensilica HiFi 4 DSP <https://www.nxp.com/docs/en/application-note/AN13970.pdf>`_ application note provides instructions for setting up the evaluation board.
+The `i.MX 8M Plus Evaluation Kit Quick Start Guide (IMX8MPLUSQSG) <https://www.nxp.com/docs/en/quick-reference-guide/8MPLUSEVKQSG.pdf>`_ contains basic information on the board and how to set it up
+In `AN13970 Running Zephyr RTOS on Cadence Tensilica HiFi 4 DSP <https://www.nxp.com/docs/en/application-note/AN13970.pdf>`_ application note provides instructions on how to launch the samples on the HiFi4 DSP.
```

Comment:
Please try as much as possible to limit the line lenght to 100 characters. the references  are exceptions
see https://github.com/OpenAMP/openamp-docs/pull/54